modules/saml/www/sp/metadata.php

```diff
@@ -77,6 +77,9 @@
 $supported_protocols = [];
 foreach ($assertionsconsumerservices as $services) {
     $acsArray = ['index' => $index];
+    if ($index == 0) {
```

Comment:
This one is problematic because not setting a default endpoint is perfectly valid in SAML.

However, if you fix my comment above, you should be able to manually set the ACS-endpoints and a set default like this;
https://simplesamlphp.org/docs/stable/simplesamlphp-metadata-endpoints#section_3